app/scripts/metamask-controller.js

```diff
@@ -2196,6 +2195,15 @@ export default class MetamaskController extends EventEmitter {
     this.extension.runtime.onMessageExternal.addListener(onMessageReceived);
     // Fire a ping message to check if other extensions are running
     checkForMultipleVersionsRunning();
+
+    if (this.onboardingController.store.getState().completedOnboarding) {
+      this.postOnboardingInitialization();
```

Comment:
Nit: if it possible to add an e2e to this?

Reply:
This is a great idea, @DDDDDanica. I added a test in commit `430c5cd50decfaa07ac344eb67c5d0ccf5e3dc1f`